ui/projects/etl-server/src/lib/pipelines/pipeline-list/pipeline-list.component.ts

```diff
@@ -47,11 +49,46 @@ export class PipelineListComponent implements OnInit {
         for (const pipeline of pipelines) {
           pipeline.display = pipeline.name;
         }
+    
         this.pipelineSections = this.processSections(pipelines, 0);
       }
     });
     this.api.queryPipelines();
   }
+  get selected() {
+    return this._selected;
+  }
+
+  set selected(value) {
```

Comment:
1. This is a lot of code duplication
2. You're not using the sort by variable
3. There's no need to call the API again, as we already have the list of pipelines here...

Reply:
what sort by variable do you mean?